astacus/coordinator/plugins/cassandra/utils.py

```diff
@@ -41,10 +41,14 @@ async def run_subop(
     return await cluster.wait_successful_results(start_results=start_results, result_class=result_class)
 
 
-async def get_schema_hash(cluster: Cluster) -> Tuple[str, str]:
+async def get_schema_hash(cluster: Cluster, exclude_nodes: Optional[list[CoordinatorNode]] = None) -> Tuple[str, str]:
```

Comment:
the pattern in other places is to specify an optional list of nodes we want this to run on, not to exclude a subset. Maybe adding a `nodes: Optional[list[CoordinatorNode]]` and pass that to run_subop?